new/LIT7049Aqweyasat.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7049Aqweyasat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለአቍያጻት፡ አንተ፡ ባርያ፡ ጸሊም፡ ጋኔን፡ ህሡም፡ </title>
```

Comment:
The wording in the incipit as supplied by the catalogue is a a little strange; 
I assume ለአቍያጻት፡ probably builds a nomilal construction with the preceding word, ወመላኪሆሙ፡. Not to break the sense (?) may be it is better to provide as a title just: ʾAnta Bāryā ṣallim gānen hǝśśum ...